### ISSUE TITLE
Installing harp under Node.js 0.12 fails

### ISSUE THREAD

**Issue by goloroden** [sintaxi/harp#403]
When I try to install harp under Node.js 0.12, it fails with the following error message:

```
> node-sass@1.2.3 install /usr/local/lib/node_modules/harp/node_modules/terraform/node_modules/node-sass
> node scripts/install.js

Binary downloaded and installed at /usr/local/lib/node_modules/harp/node_modules/terraform/node_modules/node-sass/vendor/linux-x64/binding.node

> node-sass@1.2.3 postinstall /usr/local/lib/node_modules/harp/node_modules/terraform/node_modules/node-sass
> node scripts/build.js

`linux-x64` exists; testing
module.js:355
  Module._extensions[extension](this, filename);
                               ^
Error: Module did not self-register.
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/harp/node_modules/terraform/node_modules/node-sass/lib/index.js:181:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
npm ERR! Linux 3.16.4-tinycore64
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "harp"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.0
npm ERR! code ELIFECYCLE

npm ERR! node-sass@1.2.3 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@1.2.3 postinstall script 'node scripts/build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /npm-debug.log
```

Any idea what is wrong with this?

**Comment by silentrob** [comment]
My guess is node-sass@1.2.3 is not compatible with node 0.12.

**Comment by goloroden** [comment]
You're right, the appropriate issue is [here](https://github.com/sass/node-sass/issues/653).

**Comment by sintaxi** [comment]
Hmm, thanks for catching that. We'll prepare a patch release.

**Comment by sintaxi** [comment]
Just released `v0.15.1` which reverts `node-sass to v0.9.3`. Should work on `node v0.12.0` let me know if you run into anymore issues.

**Comment by goloroden** [comment]
:-)

**Comment by valtido** [comment]
I am still getting this problem... :( I think the issue mentioned by goloroden, has already been fixed for v0.12.0 of nodejs, so maybe revert back the patch.??

**Comment by kennethormandy** [comment]
@valtido The `master` branch is compatible with Node v0.12.0 and is also using the latest version of Node-sass, we should have another release available soon.
